space-framework/pkg/space-manager/provider.go

```diff
@@ -182,21 +184,28 @@ func (p *provider) processProviderWatchEvents() {
 				space.Name = spaceName
 				space.Spec.SpaceProviderDescName = p.name
 				space.Spec.Type = spacev1alpha1apis.SpaceTypeUnmanaged
-				space.Status.SpaceConfig = event.SpaceInfo.Config
+				err := p.createSpaceSecrets(&space, event.SpaceInfo)
+				if err != nil {
+					logger.Error(err, "Failed to create secrests for space "+p.name)
+				}
 				space.Status.Phase = spacev1alpha1apis.SpacePhaseReady
-				_, err := p.c.clientset.SpaceV1alpha1().Spaces(p.nameSpace).Create(ctx, &space, v1.CreateOptions{})
+				_, err = p.c.clientset.SpaceV1alpha1().Spaces(p.nameSpace).Create(ctx, &space, metav1.CreateOptions{})
 				chkErrAndReturn(logger, err, "Detected New space. Couldn't create the corresponding Space", "space name", spaceName)
 			} else {
 				logger.V(2).Info("Updating Space object", "space", event.Name)
 				refspace.Status.Phase = spacev1alpha1apis.SpacePhaseReady
-				refspace.Status.SpaceConfig = event.SpaceInfo.Config
+
+				err := p.createSpaceSecrets(refspace, event.SpaceInfo)
+				if err != nil {
+					logger.Error(err, "Failed to create secrests for space "+p.name)
```

Comment:
Space can't be ready if we failed to create secrets.

Reply:
see above